### PR TITLE
[torch_glow] Add support for torch comparison ops

### PR DESCRIPTION
**Pull request by debayan-gh**
Summary: Adding support for torch comparison operations (eq, neq, le, lte, gt, gte)

Test Plan:
Added tests for each of the operations

### CODE REVIEW

**Comment by debayan-gh** [comment]
@jackm321

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 1]
```diff
@@ -2651,6 +2657,98 @@ Error PyTorchModelLoader::loadSqrt(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], glowNode->getResult());
 }
 
+Error PyTorchModelLoader::loadCmpEq(const torch::jit::Node *ptNode) {
```
Comment (jfix71) [inline torch_glow/src/PyTorchModelLoader.cpp:2660]: This and all the other `loadCmp*` are all the exact same except for the class used for `createNodeWithBroadcast<>` -- can we just add/use a single `loadCmp` that's templatized? We could just pass the class we're loading in the template into `createNodeWithBroadcast<>`.

Comment (debayan-gh) [reply torch_glow/src/PyTorchModelLoader.cpp]: Hi @jfix71 , Thanks for the suggestion. 
gt/ge uses lt/le nodes internally. To handle this, I have created a separate template for gt ops.

**Comment by debayan-gh** [comment]
Hi @jfix71 ,
If you have no further comments, can we pull in this patch?

**Comment by jfix71** [comment]
@jackm321 can you take a look here?

**Pull request closed without merging**